.pre-commit-config.yaml

```
@@ -84,12 +79,11 @@ repos:
     hooks:
       - id: black
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.172"
```

```suggestion
    rev: "v0.0.189"
```
This version includes the `--force-exclude` flag, see charliermarsh/ruff#1295